certified-connectors/Fleetio/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2179 @@
+                      "type": "number"
+                    },
+                    "registration_state": {
+                      "type": "number"
+                    },
+                    "trim": {
+                      "type": "string"
+                    },
+                    "vin": {
+                      "type": "string"
+                    },
+                    "year": {
+                      "type": "number"
+                    },
+                    "default_image_url_small": {
+                      "type": "string"
+                    }
+                  },
+                  "type": [
+                    "object",
+                    "null"
+                  ]
+                },
+                "vendor_id": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "vendor_name": {
+                  "type": [
+                    "null",
+                    "string"
+                  ]
+                },
+                "vendor": {
+                  "properties": {
+                    "name": {
+                      "type": "string"
+                    },
+                    "phone": {
+                      "type": "string"
+                    },
+                    "city": {
+                      "type": "string"
+                    },
+                    "region": {
+                      "type": "string"
+                    },
+                    "postal_code": {
+                      "type": "string"
+                    },
+                    "country": {
+                      "type": "string"
+                    },
+                    "external_id": {
+                      "type": "string"
+                    }
+                  },
+                  "type": [
+                    "object",
+                    "null"
+                  ]
+                },
+                "vmrs_repair_priority_class": {
+                  "properties": {
+                    "code": {
+                      "type": "string"
+                    },
+                    "color": {
+                      "type": "string"
+                    },
+                    "id": {
+                      "type": "number"
+                    },
+                    "name": {
+                      "type": "string"
+                    }
+                  },
+                  "type": [
+                    "object",
+                    "null"
+                  ]
+                },
+                "work_order_id": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "summary": "List Current Service Entries",
+        "operationId": "ListCurrentServiceEntries",
+        "parameters": [],
+        "description": "Retrieves a list of current service service",
```

Comment:
Kindly delete repeated word "service"

Reply:
@vmanoharas Thank you for the review! I went ahead and fixed the issue here, and with the apiProperties and Swagger validation checks locally to my knowledge. Let me know if any more issues arise